### DISCUSSION TITLE
Does `prettier` support `quarto`?

### DISCUSSION THREAD

**Discussion by paddyroddy** [prettier/prettier#16419] [Q&A]
`.qmd` files are essentially `.md` files but with YAML frontmatter https://quarto.org

**Comment by paddyroddy** [comment]
This blog gives some tips https://vm70.neocities.org/posts/2024-04-11-quarto-prettier

**Comment by jolars** [comment]
Here's a shameless plug for a formatter (and lsp and linter) for quarto/pandoc/rmarkdown that I've developed: https://github.com/jolars/panache
